asyncpg/protocol/prepared_stmt.pyx

```diff
@@ -38,7 +38,7 @@ cdef class PreparedStatementState:
         cdef Codec codec
 
         result = []
-        for d in self.row_desc:
+        for d in self.row_desc or ():
```

Comment:
Hm. Good catch! It would be better to check the `row_desc` and return early with an empty tuple instead:

```python
if not self.row_desc:
    return ()
```

Reply:
Makes sense! Fixed accordingly.